src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -57,4 +57,13 @@
     </Compile>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <!-- Hack to remove System.Net.Security dependency brought over by NetStandardLibrary before compile time to prevent conflicts, as this test project builds source files from that contract directly -->
```

Comment:
Do we need this hack? I thought we were doing "exclude" compile for this issue in other places.